src/unit_tests/wrappers/wazuh/wazuh_modules/wm_task_manager_wrappers.h

```diff
@@ -32,6 +32,6 @@ int __wrap_wm_task_manager_insert_task(int agent_id, const char *module, const c
 
 int __wrap_wm_task_manager_get_task_status(int agent_id, const char *module, char **status);
```

Comment:
This wrapper has changed. Remove the module from the parameters and change the name of it to `__wrap_wm_task_manager_get_upgrade_task_status`.